pkg/app/app.go

```diff
@@ -236,14 +255,20 @@ func initConfiguration(engine workflow.Engine, config configuration.Configuratio
 
 	// set default filesize threshold to 512MB
 	config.AddDefaultValue(configuration.IN_MEMORY_THRESHOLD_BYTES, configuration.StandardDefaultValueFunction(constants.SNYK_DEFAULT_IN_MEMORY_THRESHOLD_MB))
-	config.AddDefaultValue(configuration.API_URL, defaultFuncApiUrl(config, logger))
 	config.AddDefaultValue(configuration.TEMP_DIR_PATH, defaultTempDirectory(engine, config, logger))
 
+	config.AddDefaultValue(configuration.API_URL, defaultFuncApiUrl(config, logger))
+
+	err = config.AddKeyDependency(configuration.WEB_APP_URL, configuration.API_URL)
+	if err != nil {
+		logger.Print("Failed to add dependency for WEB_APP_URL:", err)
+	}
+
```

Comment:
nitpick: Maybe we should move this below line `267`? 

That would keep it consistent with the "first add dependency, then dependencies" flow I've seen above... I think?

And I wouldn't normally nitpick over these, but I feel the `initConfiguration` is already super verbose - so things could easily be overlooked when we change things a year from now and remember nothing, to avoid leaving "dangling" logic.